services/ui-src/src/components/app/App.tsx

```diff
@@ -5,14 +5,14 @@ import { Container, Divider, Flex, Heading, Stack } from "@chakra-ui/react";
 import { AppRoutes, Footer, Header, LoginCognito, LoginIDM } from "components";
 
 export const App = () => {
-  const { logout, user, userRole, showLocalLogins, loginWithIDM } = useUser();
+  const { logout, user, showLocalLogins, loginWithIDM } = useUser();
   return (
     <div id="app-wrapper">
       {user && (
         <Flex sx={sx.appLayout}>
           <Header handleLogout={logout} />
           <Container sx={sx.appContainer} data-testid="app-container">
-            <AppRoutes userRole={userRole} />
+            <AppRoutes userRole={user?.userRole} />
```

Comment:
`user?.userRole` may be undefined here, but `userRole` for the `AppRoutes` props is defined as requiring a string. This would be a type error if `useUser` was property defined to return a `UserContextInterface` instead of the `any`. Suggest removing the `any` from the return type for the `useUser` and then changing `userRole` in `AppRoutes` props to take an optional string.

Reply:
Also the `?` here is not needed as this part of the code will run only if `user` is truthy and thus must be a defined `MCRUser`